src/main/java/net/minecraftforge/event/AttachCapabilitiesEvent.java

```diff
@@ -53,8 +59,7 @@ public T getObject()
      */
     public void addCapability(ResourceLocation key, ICapabilityProvider cap)
     {
-        if (caps.containsKey(key))
-            throw new IllegalStateException("Duplicate Capability Key: " + key  + " " + cap);
```

Comment:
Unnecessary whitespace edit (unless this is an intentional part of the refactor)